src/rt-app.c

```diff
@@ -611,8 +748,8 @@ static void set_thread_affinity(thread_data_t *data, cpuset_data_t *cpu_data)
 
 	if (!CPU_EQUAL(actual_cpu_data->cpuset, data->curr_cpu_data->cpuset))
 	{
-		log_debug("[%d] setting cpu affinity to CPU(s) %s", data->ind,
-			actual_cpu_data->cpuset_str);
+		log_debug("[%d] %s setting cpu affinity to CPU(s) %s",
+			data->ind, data->name, actual_cpu_data->cpuset_str);
```

Comment:
This is not linked to fork event. Please keep it separate

Reply:
This was addressed already

https://github.com/scheduler-tools/rt-app/pull/53/commits/7fc5dc6e52c32f7abc9190ebc4e315fcb2ad74e3

Reply:
I don't understand what you want. Can you clarify please?

This was already split into its own commit

https://github.com/scheduler-tools/rt-app/pull/53/commits/7fc5dc6e52c32f7abc9190ebc4e315fcb2ad74e3

Are you looking at the latest version of the patchsets? I have to say github view of the incremental changes confuses me.

Reply:
Do you want the commit to be dropped altogether, is this is what you're saying?